### PR TITLE
Jobpost services integrated with redux stuff and added navbar for logged in pages

### PR DESCRIPTION
**Pull request by dilippoudel**

### CODE REVIEW

File: src/components/LogOut/index.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import { Button } from 'react-bootstrap'
+
+const LogOut = () => {
+  const deleteTokenFromLocalStorage = async () =>
+    await localStorage.removeItem('token')
```
Comment (Wyfy0107) [inline src/components/LogOut/index.tsx]: Please use the methods I provided inside `local-storage` folder

File: src/components/LogOut/index.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import { Button } from 'react-bootstrap'
+
+const LogOut = () => {
+  const deleteTokenFromLocalStorage = async () =>
```
Comment (Wyfy0107) [inline src/components/LogOut/index.tsx]: This function should redirect the user back to landing page also

File: src/redux/saga/employer.ts [hunk 3]
```diff
@@ -38,9 +41,29 @@ function* loginEmployerSaga() {
   }
 }
 
+function* logoutEmployerSaga() {
+  try {
+    yield localStorage.removeItem('token')
+  } catch (e) {
+    console.log(e)
+  }
+}
```
Comment (Wyfy0107) [inline src/redux/saga/employer.ts]: Why this is also here. I see you already implemented a function to do this inside `Logout` component. So either do it inside saga or in the component itself. I think it's easier to do it inside the component

**Review by Wyfy0107 — CHANGES_REQUESTED** [review]

File: src/redux/saga/employer.ts (hunk 3 above)
Comment (dilippoudel) [reply src/redux/saga/employer.ts]: this was my question in the morning's session. I implement it inside the component now.

Comment (Wyfy0107) [reply src/redux/saga/employer.ts]: Ok, then let's implement it only inside the `logout` component 👍

**Review by Wyfy0107 — APPROVED** [review]
💯  You can merge now

**Review by Tsaikkari — COMMENTED** [review]
I think the Navbar folder should be named Navbar and not Navbar-logout. There are going to be more links in the navbar, not just logout.

**Comment by Wyfy0107** [comment]
> I think the Navbar folder should be named Navbar and not Navbar-logout. There are going to be more links in the navbar, not just logout.

Sorry I merged it already, perhaps you can make an issue for that ?

**Pull request merged**